dnsapi/dns_cx.sh

```diff
@@ -82,7 +82,7 @@ existing_records() {
     return 1
   fi
   count=0
-  seg=$(printf "%s\n" "$response" | _egrep_o "{[^\{]*host\":\"$_sub_domain\"[^\}]*\}")
+  seg=$(printf "%s\n" "$response" | _egrep_o "{[^{]*host\":\"$_sub_domain\"[^}]*}")
```

Comment:
Why do you make this change ?
Is there any problem you see ?

It was added by me to make it work on freebsd and solaris.